### PR TITLE
Add a contributor cheatsheet

### PR DESCRIPTION
**Pull request by castrojo**
This started as a list of resources I would check on every day, sort of like a portal page from the 90s meets the kubectl cheat sheet. Then I started to ask around what URLs people use day-to-day and then people asked for a copy, so I figured it wouldn't be a bad thing to share: 

Contrib-ex discussion and proposal: https://groups.google.com/forum/#!topic/kubernetes-wg-contribex/-azSwMpLsZE

I think this still needs a few things to be useful to more people, perhaps someone with front end experience can make it shiny, in the meantime setting it as your new tab page in the browser has been working for me.

### CODE REVIEW

File: contributors/devel/contributor-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Kubernetes
+
+| Repo | PRs | Issues | Notes |
+| ---- | --- | ------ | ----- |
+| [k/kubernetes](https://github.com/kubernetes/kubernetes) | [PRs](https://github.com/kubernetes/kubernetes/pulls) | [Issues](https://github.com/kubernetes/kubernetes/issues) | [Meeting Notes](http://bit.ly/kubenotes)
+| [k/community](https://github.com/kubernetes/community) | [PRs](https://github.com/kubernetes/community/pulls) | [Issues](https://github.com/kubernetes/community/issues) |
+| [k/docs](https://github.com/kubernetes/kubernetes.github.io) | [PRs](https://github.com/kubernetes/kubernetes.github.io/pulls) | [Issues](https://github.com/kubernetes/kubernetes.github.io/issues)
```
Comment (cblecker) [inline contributors/devel/contributor-cheatsheet.md]: the repo was renamed kubernetes/website

File: contributors/devel/contributor-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- [Developer Guide]
+  - [Cherry Picking Guide](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md) - [Queue](http://cherrypick.k8s.io/#/queue)
+
+## SIGs and Working Groups
+
+- [SIG Master list](https://github.com/kubernetes/community/blob/master/sig-list.md)
+
+## Community
+
+- [Calendar](https://calendar.google.com/calendar/embed?src=nt2tcnbtbied3l6gi2h29slvc0%40group.calendar.google.com)
```
Comment (cblecker) [inline contributors/devel/contributor-cheatsheet.md]: Is this the right link? Calendar looks blank

**Comment by dims** [comment]
LGTM 👍

File: contributors/devel/contributor-cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Kubernetes
```
Comment (cblecker) [inline contributors/devel/contributor-cheatsheet.md]: Probably needs a clearer title :)

**Comment by idvoretskyi** [comment]
@castrojo incredible. Thank you!

File: contributors/devel/contributor-cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+- [SIG Master list](https://github.com/kubernetes/community/blob/master/sig-list.md)
+
+## Community
+
+- [Calendar](https://calendar.google.com/calendar/embed?src=nt2tcnbtbied3l6gi2h29slvc0%40group.calendar.google.com)
+- [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
+- [kubernetes-users](https://groups.google.com/forum/#!forum/kubernetes-users)
+- [StackOverflow](https://stackoverflow.com/questions/tagged/kubernetes)
+- [YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/)
```
Comment (spzala) [inline contributors/devel/contributor-cheatsheet.md]: hi @castrojo  - I guess good idea to add a slack channel link here as well. Thanks!

File: contributors/devel/contributor-cheatsheet.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- [reviewable.kubernetes.io](https://reviewable.kubernetes.io/reviews#-)
+- [Submit Queue](https://submit-queue.k8s.io)
+- [Bot commands](https://github.com/kubernetes/test-infra/blob/master/commands.md)
+- [Release Buckets](http://gcsweb.k8s.io/gcs/kubernetes-release/)
+- [Developer Guide]
+  - [Cherry Picking Guide](https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md) - [Queue](http://cherrypick.k8s.io/#/queue)
+
+## SIGs and Working Groups
+
+- [SIG Master list](https://github.com/kubernetes/community/blob/master/sig-list.md)
```
Comment (spzala) [inline contributors/devel/contributor-cheatsheet.md]: When I open the link, it says "Master SIG List" :-) - so may be good idea to use the same wording here (vs "SIG Master List") ? Also, the link is for this sub-topic so we should use this link - https://github.com/kubernetes/community/blob/master/sig-list.md#master-sig-list  And one more - since your providing sub-topics as link shouldn't we do both links - Master SIG List and "Master Working Group List" Thanks!

File: contributors/devel/contributor-cheatsheet.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+- [Calendar](https://calendar.google.com/calendar/embed?src=nt2tcnbtbied3l6gi2h29slvc0%40group.calendar.google.com)
+- [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
+- [kubernetes-users](https://groups.google.com/forum/#!forum/kubernetes-users)
+- [StackOverflow](https://stackoverflow.com/questions/tagged/kubernetes)
+- [YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/)
+
+## Tests
+
+- [What is currently testing](https://prow.k8s.io/)
```
Comment (spzala) [inline contributors/devel/contributor-cheatsheet.md]: Nit, but if you like reword it too something like "Current test status"?

File: contributors/devel/contributor-cheatsheet.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+# Kubernetes
+
+| Repo | PRs | Issues | Notes |
+| ---- | --- | ------ | ----- |
+| [k/kubernetes](https://github.com/kubernetes/kubernetes) | [PRs](https://github.com/kubernetes/kubernetes/pulls) | [Issues](https://github.com/kubernetes/kubernetes/issues) | [Meeting Notes](http://bit.ly/kubenotes)
```
Comment (spzala) [inline contributors/devel/contributor-cheatsheet.md]: Here and below two places - with "Kubernetes" header (LOC # 1) if we don't use "k/" that should be OK as it's self readable. Just a thought, it's up to you :-) Thanks!

File: contributors/devel/contributor-cheatsheet.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+| [k/community](https://github.com/kubernetes/community) | [PRs](https://github.com/kubernetes/community/pulls) | [Issues](https://github.com/kubernetes/community/issues) |
+| [k/docs](https://github.com/kubernetes/kubernetes.github.io) | [PRs](https://github.com/kubernetes/kubernetes.github.io/pulls) | [Issues](https://github.com/kubernetes/kubernetes.github.io/issues)
+
+## Workflow
+
+- [Gubernator Dashboard - k8s.reviews](https://k8s-gubernator.appspot.com/pr)
+- [reviewable.kubernetes.io](https://reviewable.kubernetes.io/reviews#-)
```
Comment (spzala) [inline contributors/devel/contributor-cheatsheet.md]: Change it to "Reviewable for Kubernetes" to be consistent with rest of the page? Another nit, if you like the suggestion.

**Comment by spzala** [comment]
@castrojo awesome and very useful. I have added few minor comments inline. Thanks!

File: contributors/devel/contributor-cheatsheet.md (hunk 7 above)
Comment (xiangpengzhao) [reply contributors/devel/contributor-cheatsheet.md]: +1 for no K

File: contributors/devel/contributor-cheatsheet.md (hunk 4 above)
Comment (xiangpengzhao) [reply contributors/devel/contributor-cheatsheet.md]: +1. Maybe also add blog/twitter, and events (if there is a link)

File: contributors/devel/contributor-cheatsheet.md (hunk 2 above)
Comment (castrojo) [reply contributors/devel/contributor-cheatsheet.md]: The events calendar just happens to be blank, I'll switch it to the contributor calendar which has all the meetings.

**Comment by castrojo** [comment]
Ok I've integrated the feedback, PTAL.

**Comment by spzala** [comment]
@castrojo thanks for addressing comments!
